# Microbiome sequencing

A collection of tools useful for 16s rRNA sequencing (not comprehensive)

# 

**General tools**

	Unix environment
	Scripting (Perl/Python/bash)
	R

**Read trimming (and adaptor removal)**

	Trimmomatic	http://www.usadellab.org/cms/?page=trimmomatic
	ConDeTri	https://github.com/linneas/condetri
	FASTX-Toolkit	http://hannonlab.cshl.edu/fastx_toolkit/
	cutadapt	https://github.com/marcelm/cutadapt
	BBDuk		https://sourceforge.net/projects/bbmap/

**Merge paired end reads**

	vsearch		https://github.com/torognes/vsearch
	usearch		http://www.drive5.com/usearch/
	PEAR		https://sco.h-its.org/exelixis/web/software/pear/
	FLASh 		http://ccb.jhu.edu/software/FLASH/
	MeFiT		 https://github.com/nisheth/MeFiT

**Quality filtering**

	vsearch https://github.com/torognes/vsearch
	usearch	http://www.drive5.com/usearch/
	MeFiT	https://github.com/nisheth/MeFiT
	
**Chimera discovery**

	vsearch https://github.com/torognes/vsearch
	usearch http://www.drive5.com/usearch/

**Dereplication/OTU clustering/Taxonomic assignments**
	
	vsearch https://github.com/torognes/vsearch
	usearch http://www.drive5.com/usearch/
	mothur	https://www.mothur.org/
	QIIME2	https://qiime2.org/
	
**Phylogenetic tree reconstruction**

	FastTree	http://www.microbesonline.org/fasttree/
	
**Downstream analysis (e.g. alpha/beta diversity, indicator species)**

	phyloseq		http://joey711.github.io/phyloseq/
	mothur			https://www.mothur.org/
	vegan			https://cran.r-project.org/web/packages/vegan/index.html
	indicspecies	https://cran.r-project.org/web/packages/indicspecies/
	DESeq2			http://bioconductor.org/packages/release/bioc/html/DESeq2.html
	ALDEx2			https://bioconductor.org/packages/release/bioc/html/ALDEx2.html
	QIIME2			https://qiime2.org/
	metagenomeSeq	http://cbcb.umd.edu/software/metagenomeSeq/
	microbiome		https://github.com/microbiome/microbiome/
	
**Pipeline platforms (interactive)**

	mothur	https://www.mothur.org/
	QIIME2	https://qiime2.org/
	MEGAN	http://joey711.github.io/phyloseq/
	
